### ISSUE TITLE
React 15.2.0 warns about xmlns

### ISSUE THREAD

**Issue by aaronjensen** [boopathi/react-svg-loader#44]
Thanks to https://gist.github.com/jimfb/d99e0678e9da715ccf6454961ef04d1b, react is now warning about adding the `xmlns` prop to `svg` elements. Obviously, one could remove the xmlns attribute from their svgs, but this causes problems if the same svgs are used as images in other places. It'd be best if this loader stripped the attributes react doesn't like imo.

**Comment by aaronjensen** [comment]
In the meantime, this is a workaround:

```js
       {
         test: /icons\/.*\.svg$/,
         loader: 'react-svg',
         query: {
           es5: true,
           svgo: {
             plugins: [{ removeAttrs: { attrs: 'xmlns.*' } }],
           },
         },
       },
```

**Comment by boopathi** [comment]
Thanks!.

I intentionally keep them in the file thinking it might be necessary / useful. - https://github.com/boopathi/react-svg-loader/blob/25a166c8411be42a3e99a44e8c7aa6e7396d7637/src/plugin.js#L10-L16 

PR Welcome 😃

**Comment by tonygentilcore** [comment]
FYI, this was fixed in 15.3.0:
https://github.com/facebook/react/blob/master/CHANGELOG.md#react-dom-1

**Comment by aaronjensen** [comment]
indeed, I'll close this.
